Formula/c-toxcore.rb

```diff
@@ -0,0 +1,38 @@
+class CToxcore < Formula
```

Comment:
Everyone else seems to call this `toxcore` or `libtoxcore`. We should probably choose one of those names instead. See:

- https://repology.org/project/toxcore/versions
- https://repology.org/projects/?search=toxcore

Reply:
Makes sense, will do - I was concerned about naming it `toxcore` due to possible confusion with the fork, but now that we've added the comment about the fork & why it wasn't chosen I feel better about it!